responsibleai/tests/causal/test_causal_manager.py

```diff
@@ -60,7 +61,7 @@ def test_causal_save_and_load(self, housing_data, tmpdir):
             dm = DirectoryManager(parent_directory_path=save_dir,
                                   sub_directory_name=causal_dir)
             causal_analysis_pkl_file_path = \
-                dm.get_data_directory() / "causal_analysis.pkl"
+                dm.get_data_directory() / ("causal_analysis" + FileFormats.PKL)
```

Comment:
Oh I see, you are moving common ones in a file. These could be a const in their own file. May be not for this PR.